Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -263,14 +286,28 @@ public struct TestRequirement: Decodable {
         if let topologies = self.topologies {
             // When matching a "sharded" topology, test runners MUST accept any type of sharded cluster (i.e. "sharded"
             // implies "sharded-replicaset", but not vice versa).
-            if topology == .shardedReplicaSet && topologies.contains(.sharded) {
```

Comment:
since mongo-orchestration always sets up a `.shardedReplicaSet` (see [configs](https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/orchestration/configs/sharded_clusters)) I think this evaluates to true basically anytime we test against a sharded cluster and `sharded` is among the topologies to run on.... I'm guessing we've maybe just gotten lucky and none of the tests have had extra requirements beyond that? we don't actually run many tests with the unified runner yet